packages/patterns-react/src/patterns/ListSection/ListSection.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import classNames from 'classnames';
+import { featureFlag } from '@carbon/ibmdotcom-utilities';
+import { LISTSECTION } from '../../internal/FeatureFlags';
+import ListGroup from './ListGroup';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * List Section pattern
+ *
+ * @param {object} props props object
+ * @param {string} props.title List section title
+ * @param {string} props.copy List section  short copy to support the title
+ * @param {object} props.listGroup variation of the List section standard, standard with jump link and standard with card link
+ * @returns {object} JSX Object
+ */
+const ListSection = ({ title, copy, listGroup }) =>
+  featureFlag(
+    LISTSECTION,
+    <section
+      data-autoid={`${stablePrefix}--listsection`}
+      className={classNames(`${prefix}--listsection`)}>
+      <div className={`${prefix}--listsection__container`}>
+        <div className={`${prefix}--listsection__row`}>
+          <div className={`${prefix}--listsection__col`}>
+            <h1 className={`${prefix}--listsection__title`}>{title}</h1>
+            <div className={`${prefix}--listsection__content`}>{copy}</div>
+          </div>
+          <div className={`${prefix}--listsection__listgroup__col`}>
+            {renderListGroup(listGroup)}
+          </div>
+          <div className={`${prefix}--listsection__divider__col`}>
+            <div className={`${prefix}--listsection__divider`}></div>
+          </div>
+        </div>
+      </div>
+    </section>
+  );
+
+/**
+ * Render List Group Component
+ *
+ * @param {object} listGroupItems listGroupItems Object
+ * @returns {object} JSX Object
+ */
+const renderListGroup = listGroupItems => {
```

Comment:
Should be a private method, named with underscore `_renderListGroup` and `@private` in jsdocs.

Reply:
will do.

Reply:
marked renderListGroup function as private by [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/573/commits/6b28d40fcd3278749c38a30d336f80c5f19b5a11)